ksql-engine/src/test/java/io/confluent/ksql/schema/registry/KsqlSchemaRegistryClientFactoryTest.java

```diff
@@ -41,6 +43,7 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.notNullValue;
+import static org.testng.AssertJUnit.assertEquals;
```

Comment:
did you mean to use testng or junit assertEquals?

Reply:
Good catch. The `JUnit` in that name did me in.